app/models/fe/state_chooser.rb

```diff
@@ -3,8 +3,8 @@
 # - drop down of states
 module Fe
   class StateChooser < Question
-    def choices(country = 'US')
-      @states = Carmen.states(country)
+    def choices(locale = nil, country = 'US')
```

Comment:
Me too, which is obviously not what we want. Is there another code change to wherever StateChooser#choices is being used to make sure it's being called properly now that there are 2 params?